extensions/azurePublishNew/src/components/provisioningWizards/steps/CreateActionContentStep.tsx

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import * as React from 'react';
+import formatMessage from 'format-message';
+import { Text } from 'office-ui-fabric-react/lib/Text';
+import styled from '@emotion/styled';
+import { FluentTheme } from '@uifabric/fluent-theme';
+import { Link, Stack } from 'office-ui-fabric-react';
+
+export const Content = styled(Stack)`
```

Comment:
why are the styled components below are exported?
if you want to reuse them, move them outside of this file into a common file. (do this cause you are redefining the exact same components in other files)

Reply:
They were being used by other components, I will move them to a common file.